src/features/user/service/utils.ts

```diff
@@ -0,0 +1,26 @@
+export const getCurrentYear = () => {
+  const today = new Date();
+  return today.getFullYear();
+};
+
+export const getCurrentMonth = () => {
+  const today = new Date();
+  return today.getMonth() + 1;
+};
+
+export const getLastDayOfMonth = (): number => {
+  const targetYear = getCurrentYear();
+  const targetMonth = getCurrentMonth();
+  return new Date(targetYear, targetMonth, 0).getDate();
+};
+
+export const getDayFromDateString = (dateString: string) => {
+  const date = new Date(dateString);
+  const day = date.getDate();
+  return day;
+};
```

Comment:
```suggestion
export const getDayFromDate = (date: string | Date) => {
  const date = new Date(date);
  const day = date.getDate();
  return day;
};
```
Date 객체는 문자열도 되고 Date 객체를 받더라도 동일하게 동작합니다
범용성을 위해 두 타입을 모두 받는게 어떨까요?
```ts
const date = new Date('2024-11-01');

date; // Fri Nov 01 2024 09:00:00 GMT+0900 (한국 표준시)

new Date(date); // Fri Nov 01 2024 09:00:00 GMT+0900 (한국 표준시)

new Date(new Date('2024-11-01')); // Fri Nov 01 2024 09:00:00 GMT+0900 (한국 표준시)
```

Reply:
추가적으로 dayjs와 같은 날짜 관련 라이브러리를 사용하지 않는 이유가 있을까요? 😄

Reply:
> Date 객체는 문자열도 되고 Date 객체를 받더라도 동일하게 동작합니다
범용성을 위해 두 타입을 모두 받는게 어떨까요?

좋은 방법인 것 같습니다 !!👍👍👍

> 추가적으로 dayjs와 같은 날짜 관련 라이브러리를 사용하지 않는 이유가 있을까요? 😄

사실 이런 라이브러리가 있는지 먼저 찾아보지를 않아서 몰랐습니다 🙄

Date를 다루는것이 해당 출석체크 정도와 가입일을 표시하는 정도이고 그리고 이런 유틸 함수를 만드는데 엄청 어려운 작업은 아니여서   물론 dayjs같은 추가적인 라이브러리를 설치하면 공식문서를 읽고 활용하는데 좀더 오래걸릴 것 같다는 생각이 들어서 추가적으로 라이브러리를 설치하지는 않았습니다